parser/SqlParser.ypp

```diff
@@ -759,6 +780,51 @@ opt_table_constraint_def_commalist:
     /* $$ = $1; */
   };
 
+opt_column_list:
+  /* empty */
```

Comment:
Sure, will modify these to return NULL, though I was also tempted to use the %empty directive to specify empty rules, but this feature is not backward-compatible with bison 2.5 ( the one that is run on Travis builds)